neonvm/controllers/virtualmachine_controller.go

```diff
@@ -1165,6 +1165,10 @@ func podSpec(virtualmachine *vmv1.VirtualMachine) (*corev1.Pod, error) {
 		})
 	}
 
+	if virtualmachine.Spec.Guest.AdditionalKernelCommandLine != nil {
+		pod.Spec.Containers[0].Args = append(pod.Spec.Containers[0].Args, "-additionalKernelCommandLine="+*virtualmachine.Spec.Guest.AdditionalKernelCommandLine)
```

Comment:
```suggestion
		pod.Spec.Containers[0].Args = append(pod.Spec.Containers[0].Args, fmt.Sprintf("-additionalKernelCommandLine=%s", *virtualmachine.Spec.Guest.AdditionalKernelCommandLine))
```

In general, `fmt.Sprint[f]` is faster than string addition because it uses a buffer pool, but string addition does not.